### PR TITLE
(DOCSP-30990): Swift: Add docs for session multiplexing and sync timeout

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-30990

### Staged Changes

- [Connect to an App Services App](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-30990/sdk/swift/app-services/connect-to-app-services-backend/)
- [Manage Sync Sessions](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-30990/sdk/swift/sync/sync-session/)
- [Sync Data](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-30990/sdk/swift/sync/#sync-connection-and-timeout-behaviors)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/app-services/connect-to-app-services-backend.txt [hunk 1]
```diff
@@ -41,6 +41,38 @@ You can pass a configuration object to ``App``:
...
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 10.41.0
+
+You can set the ``enableSessionMultiplexing`` bool on the 
+:swift-sdk:`AppConfiguration <Extensions/AppConfiguration.html>` to specify
+whether the Realm Swift SDK should open a connection to the server for each 
+synced realm, or share a connection to the server for all synced realms.
+
+If you do no specify a value for this bool, Realm defaults to sharing a
```
Comment (cbullinger) [inline source/sdk/swift/app-services/connect-to-app-services-backend.txt:54]:
```suggestion
If you do not specify a value for this bool, Realm defaults to sharing a 
```

File: source/sdk/swift/app-services/connect-to-app-services-backend.txt [hunk 2]
```diff
@@ -41,6 +41,38 @@ You can pass a configuration object to ``App``:
...
+
+.. literalinclude:: /examples/generated/code/start/RealmApp.snippet.app-config-enable-session-multiplexing.swift
+   :language: swift
+
+Sync Timeout Options
+~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 10.41.0
+
+You can set various ``SyncTimeout`` options on the
```
Comment (cbullinger) [inline source/sdk/swift/app-services/connect-to-app-services-backend.txt:65]:
```suggestion
You can set various sync timeout options on the 
```
is there a ``SyncTimeout``?

**Review by cbullinger — APPROVED** [review]
Overall LGTM

File: source/sdk/swift/app-services/connect-to-app-services-backend.txt (hunk 2 above)
Comment (dacharyc) [reply source/sdk/swift/app-services/connect-to-app-services-backend.txt:65]: Nope, good catch!

**Pull request merged**